app/assets/stylesheets/front/pages/meditations.sass

```diff
@@ -1,132 +1,77 @@
 @import "../../mixins/vars"
```

Comment:
This file is bloated and needs to be divided into several files. I will do that in a subsequent commit.